kuma/static/layout-tests/no-inline-js.html

```diff
@@ -0,0 +1,1438 @@
+
+
+
+<!DOCTYPE html>
+<html lang="en-US" dir="ltr" class="no-js">
+<head prefix="og: http://ogp.me/ns#">
+  <meta charset="utf-8">
+
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge">
+  <script>(function(d) { d.className = d.className.replace(/\bno-js/, ''); })(document.documentElement);</script>
+  <title>Array.prototype.forEach() - JavaScript | MDN</title>
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="robots" content="index, follow">
+
+  
+<link rel="preload" href="https://developer.mozilla.org/static/fonts/locales/ZillaSlab-Regular.subset.bbc33fb47cf6.woff2" as="font" type="font/woff2" crossorigin />
```

Comment:
There's a risk that the hashed versions of these assets will not be available. We don't keep old versions of assets, but the CDN probably does. I'd say this is a low risk, but look for it in speedcurve. The same name without the hash will give the latest asset (``https://developer.mozilla.org/static/fonts/locales/ZillaSlab-Regular.subset.woff2``) with a 7-day cache timeout.

Reply:
Do you recommend dropping the has before deploying?

Reply:
I'm not sure. Will the 7-day cache cause issues in speedcurve (continue using hashed, update if it breaks), or does it always request fresh (use unhashed and get latest)?